frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -377,6 +377,14 @@ interactivetour-InteractiveTour--history-content =
     each entry indicate its review status (Approved, Rejected or
     Unreviewed).
 
+interactivetour-InteractiveTour--terms-title = Terms
+interactivetour-InteractiveTour--terms-content =
+    Test
```

Comment:
_The Terms panel contains any specialized words found in the source string, along with their definition, usage example, part of speech and translation. By clicking on the term, its translation gets inserted into the editor._